config/v1/types_features.go

```diff
@@ -44,22 +47,22 @@ type FeaturesList struct {
 	Items           []Features `json:"items"`
 }
 
-var (
-	// TechPreviewEnabledKubeFeatures  is a list of features to turn on when TechPreviewNoUpgrade is turned on.  Only the exceptions
-	// are listed here.  The feature gates left in their default state not listed.
-	TechPreviewEnabledKubeFeatures = []string{}
-
-	// TechPreviewDisabledKubeFeatures is a list of features to turn on when TechPreviewNoUpgrade is turned on.  Only the exceptions
-	// are listed here.  The feature gates left in their default state not listed.
-	TechPreviewDisabledKubeFeatures = []string{}
-
-	// DefaultEnabledKubeFeatures is a list of features to turn on when the default featureset is turned on.  Only the exceptions
-	// are listed here.  The feature gates left in their default state not listed.
-	DefaultEnabledKubeFeatures = []string{}
+type FeatureEnabledDisabled struct {
+	Enabled  []string
+	Disabled []string
+}
 
-	// DefaultDisabledKubeFeatures is a list of features to turn off when the default featureset is turned on.  Only the exceptions
-	// are listed here.  The feature gates left in their default state not listed.
-	DefaultDisabledKubeFeatures = []string{
-		"PersistentLocalVolumes", // disable local volumes for 4.0, owned by sig-storage/hekumar@redhat.com
-	}
-)
+// FeatureSets Contains a map of Feature names to Enabled/Disabled Features.
+// NOTE: The caller needs to make sure to check for the existence of the value
```

Comment:
Explain the case where it won't exist.  Upgrades with new enumation values will be a common one.

Reply:
Added in 645f70d